data/human/human missions.txt

```diff
@@ -107,7 +107,269 @@ mission "Pookie, Part 2"
 		payment 80000
 		dialog "In the past few days, Pookie has barked incessantly and thrown up several times despite being fed perfectly on schedule, and the room you've been keeping her in will likely smell like dog urine for months. You are all too happy to return her to her owner, who pays you <payment>."
 		log "Minor People" "Pocahontas (Pookie)" `A dog...`
-		
+
+
+
+mission "Pookie, Part 3"
+	name "Stray Dog"
+	description `Find a home for a stray dog you picked up on <origin>. She might have been from one of the Paradise Worlds.`
+	source "Freedom"
+	passengers 1
+	to offer
+		has "Pookie, Part 2: done"
+	on offer
+		conversation
+			`The weather on <origin> is unusually rough today; the sky ripples with thunder while rain whips against locals brave or desperate enough to stay outside. Through the mist, you see a building with neon lettering saying, "East Mesa Indoor Racing Complex: Bets are open!"`
+			choice
+				`	(Check it out.)`
+				`	(Stay dry in your ship.)`
+					defer
```

Comment:
Instead of immediately deferring after this choice, you should probably instead lead into a short line about you returning to your ship and then end the mission, instead of having this choice end the mission right there, since it doesn't seem to me like a choice that if I chose it would end the mission.